deploy/sdk/src/dynamo/sdk/request_tracing.py

```
@@ -0,0 +1,372 @@
+
+        @wraps(func)
+        async def wrapper(self, *args, **kwargs):
+            # Check if the function has the request ID parameter
+            if param_name in sig.parameters:
+                # Extract the request ID parameter from kwargs or args
+                request_id = None
+                extracted_from_args = False
+
+                if param_name in kwargs:
+                    request_id = kwargs[param_name]
+                    extracted_from_args = True
```

Ruff (0.11.9)

Local variable `extracted_from_args` is assigned to but never used

Remove assignment to unused variable `extracted_from_args`

(F841)